### ISSUE TITLE
[Question] AIRL discriminator / generator loss function

### ISSUE THREAD

**Issue by kwk2696** [HumanCompatibleAI/imitation#451]
Hello, I'm currently using the AIRL to train the mujoco environments, but I could not get the good results ...

I'm quite confused about the loss function of the discriminator in AIRL.

Following the original paper, the loss function of discriminator is defined as
 - -log_sigmoid(-logits_pi) - log_sigmoid(logits_exp) 

However, it is defined as binary_cross_entropy of log_pi - f(s,a) in the implementation. 
I understood the purpose of using binary_cross_entropy loss, since it assigns low value for generator's trajectories and high value for expert's trajectories.

Also, reward for generator (PPO policy) is defined as f(s,a) - log_pi in the paper. 
However, in the implementation (RewardVecEnvWrapper) the reward is given as f(s,a).

Is there is any reason for defining loss functions differently?
And is there any plan for benchmark AIRL on mujoco environments?
